packages/amplify-category-notifications/src/auth-helper.ts

```diff
@@ -117,37 +124,38 @@ async function checkAuth(context, resourceName) {
   }
 
   if (checkResult.errors && checkResult.errors.length > 0) {
-    context.print.warning(checkResult.errors.join(os.EOL));
+    printer.warn(checkResult.errors.join(os.EOL));
```

Comment:
just curious; we are moving away from using the print command in $TSContext?

Reply:
yes, `context.print` is deprecated